clientupdate/clientupdate_windows.go

```diff
@@ -26,3 +33,25 @@ const certSubjectTailscale = "Tailscale Inc."
 func verifyTailscale(path string) error {
 	return authenticode.Verify(path, certSubjectTailscale)
 }
+
+func launchTailscaleAsGUIUser(exePath string) error {
```

Comment:
Is there a reason why you're using this instead of `winutil.StartProcessAsChild`?

Reply:
I tried using `StartProcessAsCurrentGUIUser` at first, which failed calling `windows.GetShellWindow` via `GetDesktopPID` (returning `ErrNoShell`).
I can try using `StartProcessAsChild`,  but what parent PID should I use?